metadata.json

```diff
@@ -26,7 +26,7 @@
   "requirements": [
     {
       "name": "pe",
-      "version_requirement": ">=3.7.0 <4.0.0"
+      "version_requirement": ">=3.0.0 <=4.3.1"
```

Comment:
Yes the Ruby handling is the only issue to be Puppet 4 compatible.  I'm using my patch in my environment (Puppet Enterprise 2015.2) for over a month and everything is working smooth.

The way Ruby is installed for version 3.0.0 up to version 3.7 has not changed...